src/luminol/modules/time_series.py

```diff
@@ -43,7 +45,7 @@ def timestamps_ms(self):
         """
         Return list of timestamp values in order by milliseconds since epoch.
         """
-        return map(lambda ts: ts * 1000, self.timestamps)
+        return list(map(lambda ts: ts * 1000, self.timestamps))
```

Comment:
As before, list comprehension may be more readable: `[ts * 1000 for ts in self.timestamps]`

Reply:
Agreed.